conda_recipes/cinema4d-openjd/recipe/recipe.yaml

```diff
@@ -0,0 +1,51 @@
+# Recipe in rattler-build format.
+# See https://prefix-dev.github.io/rattler-build/latest/
+
+context:
+  version: "0.4.1"
+
+package:
+  name: cinema4d-openjd
+  version: ${{ version }}
+
+# This source reference uses the source archive published to PyPI
+source:
+  url: https://pypi.io/packages/source/d/deadline-cloud-for-cinema-4d/deadline_cloud_for_cinema_4d-${{ version }}.tar.gz
+  sha256: 0404af9f6740a9778ced6f698c096ef52f6e8a0a73f975c4520727578a3dfabb
+  patches:
+    - 0001-Remove-version-hook.patch
+
+build:
+  number: 0
+  script:
+    - python -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python
+    - hatchling
+    - hatch-vcs
+    - pip
+  run:
+    - python
+    - deadline 0.48.*
+    - openjd-adaptor-runtime 0.8.*
```

Comment:
Do these packages need to be in the conda channel for this to work? Or do they get pulled from pip automatically? Or maybe conda has a copy of all PyPi libraries? I don't think we've published the Deadline library to conda forge.

Reply:
> Do these packages need to be in the conda channel for this to work?

Customers would have to also build the `deadline` and `openjd-adaptor-runtime` packages before building this package. These recipes are in the https://github.com/aws-deadline/deadline-cloud-samples/tree/mainline/conda_recipes. Maybe it makes more sense to add a readme so that its clearer?